vue-model-api/.eslintrc.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
   plugins: ["@typescript-eslint"],
   rules: {
     "@typescript-eslint/no-unused-vars": [
-      "error",
+      "warn",
```

Comment:
This should be reverted.
(n general `vue-model-api` has some dead code, that we should get rid of before merging.)